### ISSUE TITLE
merge from / populate from : object support

### ISSUE THREAD

**Issue by garanews** [MISP/MISP#3115]
It will be nice to have the object and relationships support when doing merge/populate from  between events.
At the moment these 2 features are supported only at attribute level.

**Comment by iglocska** [comment]
ping @RichieB2B ;) Do you guys still use this feature?

Also, we'll have the whole extended event concept coming up soon (tm) that will supersede it.

**Comment by RichieB2B** [comment]
Yes, we still use it regularly. The extended event concept sounds promising!

**Comment by cvandeplas** [comment]
I can confirm that there is still no object support when merging events.
